[2023-12-12 14:05:09] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn 8f5e3f1dc112525a 127.0.0.1:63783 failed checkpoint posthandshake: Connected to self
[2023-12-12 14:05:09] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn 8f5e3f1dc112525a 127.0.0.1:20131 failed checkpoint posthandshake: Connected to self
[2023-12-12 14:05:39] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn 8f5e3f1dc112525a 127.0.0.1:63920 failed checkpoint posthandshake: Connected to self
[2023-12-12 14:05:39] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn 8f5e3f1dc112525a 127.0.0.1:20131 failed checkpoint posthandshake: Connected to self
[2023-12-12 14:05:53] [warning] request.go:11 dpchain/ginHttp/pkg/app.MarkErrors Http: OpenTxCheckMode Validation is failed -- Param:commandStr , ErrMsg:on:开启交易上链检查 off:关闭交易上链检查

